package.json

```diff
@@ -34,6 +34,7 @@
     "@yarnpkg/core": "^2.0.0-rc.29",
     "@yarnpkg/lockfile": "^1.1.0",
     "event-loop-spinner": "^2.0.0",
+    "got": "11.4.0",
```

Comment:
Pin got version to 11.4.0 to fix bug with types in transitive dep

https://github.com/sindresorhus/got/issues/1350

Reply:
Can we force this version and up?

Reply:
What do you mean?